apps/love-producer/values-tucson-teststand.yaml

```diff
@@ -102,6 +102,9 @@ producers:
   dsm2:
     name_index: DSM:2 --log-level 10
     WEBSOCKET_HOST: love-nginx-service/love/manager/producers/ws/subscription
+  eascsc:
```

Comment:
You can call it `eas` here since this deployment is isolated in the `love` namespace.